menas/ui/components/schema/schemaDetail.view.xml

```diff
@@ -56,6 +56,22 @@
                         </u:FileUploader>
                         <Button text="Upload File" press="handleUploadPress"/>
                     </IconTabFilter>
+                    <IconTabFilter id="LoadRemote" icon="sap-icon://internet-browser" text="Load from URL">
```

Comment:
I have been thinking about the usability part here. I am not sure I like a new tab. Can we put it in `Upload New` and have both options there? Something like:
![image](https://user-images.githubusercontent.com/17467768/78144574-13f07780-7430-11ea-9675-6a6eece3e016.png)
(excuse the quality)

Reply:
This can certainly be done. In this case, do we want to duplicate the schemaType select?

Should we, then?

Reply:
I would duplicate it for now. If anyone has a better UX proposition I am happy to listen.

Reply:
Maybe in this case, we can remove the schema type from the bottom part and just add a label saying "Get from Schema Registry" or something? And it will always imply Avro for the URL for now.

We can keep the support of other formats in the backend part for the future. So far we don't support it yet anyway (e.g. we don't have StructType and Copybook schema registries).

Reply:
OK, will change it as suggested.

Reply:
![upload-remote-load-in-one](https://user-images.githubusercontent.com/4457378/78272550-f1cd2700-750d-11ea-9327-e8d2550fd51f.png)
The only downside is that I had to set the width of the `HBox`es to `35em` because I don't know how to make the FileUploader stretchable (it seems that there is no support for that).